[C#][Bot-Solutions] Error NU5048: The 'PackageIconUrl'/'iconUrl' element is deprecated

#### What project is affected?
Bot-Solutions

#### What language is this in?
C#

#### What happens?
Having an Azure Pipeline configured to build the C# Bot-Solutions, the execution is stopped with an error during compilation.

#### What are the steps to reproduce this issue?
1. Create a new Azure Pipeline with steps
1. Add `Use NuGet 4.9.1` task
1. Add `dotnet restore` task
1. Add `dotnet build` test
1. Execute the Azure Pipeline pointing to `master` branch
1. Check that the compilation is stopped because an error of [NU5048](https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5048)

#### What were you expecting to happen?
The compilation need to finish correctly without any issue

#### Can you share any logs, error output, etc.?
```bash
##[error]C:\Program Files\dotnet\sdk\3.1.402\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets(198,5): Error NU5048: The 'PackageIconUrl'/'iconUrl' element is deprecated. Consider using the 'PackageIcon'/'icon' element instead. Learn more at https://aka.ms/deprecateIconUrl
```

#### Any screenshots or additional context?
![image](https://user-images.githubusercontent.com/11904023/96733466-dd5c5900-138f-11eb-90d0-a80464fb94fb.png)

--- a/sdk/csharp/Directory.Build.props
+++ b/sdk/csharp/Directory.Build.props
@@ -53,6 +53,6 @@
       Suppress a warning about upcoming deprecation of PackageLicenseUrl. When embedding licenses are supported,
       replace PackageLicenseUrl with PackageLicenseExpression.
     -->
-    <NoWarn>$(NoWarn);NU5125;NU1701;8002</NoWarn>
+    <NoWarn>$(NoWarn);NU5125;NU1701;8002;NU5048</NoWarn>
   </PropertyGroup>
 </Project>

